master.css

```diff
@@ -13,6 +13,18 @@ body {
 }
 table {
     margin-bottom: 2em; width: 100%;
+    border-spacing: 0;
+}
+thead {
+    position: -webkit-sticky;
+    position: -moz-sticky;
+    position: -ms-sticky;
+    position: -o-sticky;
```

Comment:
@kangax: Yes, FF doesn't seem to support sticky for table elements ( http://jsbin.com/sejoqapoda/1/edit )